packages/celer-web-app/src/core/settings/MapDisplayMode.ts

```diff
@@ -32,6 +24,25 @@ export const MapDisplayModes = {
 		mapSize: 0,
 		next: ()=>MapDisplayModes.Auto
 	},
-} as ReadonlyMapOf<MapDisplayMode>;
+};
 
-export const MapDisplayModeStorage = new SettingStorage("MapDisplayMode", MapDisplayModes, MapDisplayModes.Auto);
+export const getNextMap = (map: string) =>
+{
+	if(map == "Auto")
+	{
+		return MapDisplayModes.Auto;
+	}
+	if(map == "Wide")
+	{
+		return MapDisplayModes.Wide;
+	}
+	if(map == "Half")
+	{
+		return MapDisplayModes.Half;
+	}
+	if(map == "Narrow")
+	{
+		return MapDisplayModes.Narrow;
+	}
+	return MapDisplayModes.Hidden;
+};
```

Comment:
Wait.. now I'm really confused. The function is `getNextMap` but what you are doing is essentially `MapDisplayModes[map]` Again if this is intentionally please explain in a comment.